src/main/webapp/app/exercises/shared/result/result.component.ts

```diff
@@ -210,6 +210,8 @@ export class ResultComponent implements OnInit, OnChanges {
             // When the result was building and is not building anymore, we evaluate the result status.
         } else if (changes.isBuilding && changes.isBuilding.previousValue && !changes.isBuilding.currentValue) {
             this.evaluate();
+        } else {
+            console.log(changes);
```

Comment:
Do you want to keep this in production or is this only temporary?

Reply:
Dominik found an issue and I'm trying to investigate...